website/docs/index.html.markdown

```diff
@@ -3,19 +3,19 @@ layout: "ovh"
 page_title: "Provider: OVH"
 sidebar_current: "docs-ovh-index"
 description: |-
-  The OVH provider is used to interact with the many resources supported by OVH. The provider needs to be configured with the proper credentials before it can be used.
+  The OVH provider is used to interact with the many resources supported by OVHcloud. The provider needs to be configured with the proper credentials before it can be used.
 ---
 
-# OVH Provider
+# OVHcloud Provider
 
-The OVH provider is used to interact with the many resources supported by OVH. 
+The OVHcloud provider is used to interact with the many resources supported by OVHcloud. 
 The provider needs to be configured with the proper credentials before it can be used.
```

Comment:
the OVH provider is used to interact with the many resources supported by OVHcloud.